source/releases/lts.html.erb

```diff
@@ -0,0 +1,105 @@
+      <ul>
+        <li>
+          <p>
+            Because each LTS release will receive security fixes for 60 weeks, we will backport them to at least the two most recent LTS releases. <a href="http://emberjs.com/security/">Read more aboutEmber's security policy</a>.
+          </p>
+        </li>
+        <li>
+          <p>
+            Release notes for each LTS will contain a summary of features and deprecations added since the previous LTS. This will likely consist of a number of links to documentation, guides, and release notes for stable releases.
+          </p>
+        </li>
+        <li>
+          <p>Addons and applications should constrain themselves to using Ember's public APIs. However, in practice we recognize that experimentation and use of "intimate" APIs is a reality. To ensure addons and apps using these intimate APIs have a migration
+            path forward, any change to a heavily used private APIs will receive a deprecation warning in at least one LTS release. This gives LTS users and the addon community about six months to migrate away. For example, the <code>view</code> APIs (private
+            in Ember 2.x) will be marked as deprecated in Ember 2.4 LTS, and removed before Ember 2.8 LTS.
+          </p>
+        </li>
+        <li>
+          <p>We encourage addon authors to maintain support for the latest LTS release, in addition to tracking changes in the stable releases.</p>
+        </li>
+      </ul>
+      <p>In practice, since these releases still abide by <a href="http://semver.org/">semantic versioning</a>, upgrading from LTS release to LTS release should not be significantly more work than upgrading along the six-week release cycle.</p>
+      <p>Upgrading less frequently will mean, of course, that a developer will wait longer to take advantage of new features. Additionally, it means the changes in best practices between releases (and in the number of deprecations) may feel more significant.</p>
+    </section>
+    <section>
+      <h3 class="anchorable-toc" id="toc_legacy-view-and-controller-addons"><a class="toc-anchor" href="#toc_legacy-view-and-controller-addons"></a>Legacy View and Controller Addons</h3>
+      <p>
+        Ember has provided the <a href="https://github.com/emberjs/ember-legacy-views">ember-legacy-views</a> and <a href="https://github.com/emberjs/ember-legacy-controllers">ember-legacy-controllers</a> addons since Ember 1.13. For large codebases, these
+        addons provide a slower off-ramp for legacy view and controller features. Big apps can move to Ember 2.0 without the complete removal of views.
+      </p>
+      <p>
+        These addons rely on several private APIs expected to undergo significant refactoring in Ember 2.5 or 2.6. Therefore, the Ember 2.4 LTS will likely be the last LTS to support these addons. They will become unsupported in September or October with
+        the release of Ember 2.8 LTS.
+      </p>
+    </section>
+    <section>
```

Comment:
Since this section is only relevant in the blog post, it can be removed from the LTS page. Thank you! 

- [ ] remove this section

Reply:
Removed!